src/mpi/coll/ibcast/ibcast_utils.c

```diff
@@ -54,7 +54,6 @@ int MPII_Ibcast_sched_add_length(MPIR_Comm * comm, int tag, void *state)
     return mpi_errno;
 }
 
-
```

Comment:
We do not remove empty line if we are not changing the code around it.

Reply:
fixed.